docs/CONTRIBUTING.md

```diff
@@ -56,18 +56,41 @@ Vector team member will find this document useful.
 
 ### New sources, sinks, and transforms
 
-If you're contributing a new source, sink, or transform to Vector, thank you that's way cool! There's a few steps you
-need to think about if you want to make sure we can merge your contribution. We're here to help you along with these steps,
-but they are a blocker to getting a new integration released.
-
-To merge a new source, sink, or transform, you need to:
+If you're thinking of contributing a new source, sink, or transform to Vector, thank you that's way cool! There's a few
+steps you need to think about if you want to make sure we can merge your contribution. We're here to help you along with
```

Comment:
nit- "your contribution" makes it sound like a PR, which contradicts the next paragraph.

Reply:
Mhmm, unchanged from the original text - but that's fair. Happy to change it if you'd like.

Reply:
I guess the contradiction could be avoided by putting the new Q&A in a sub-heading, or could just remove the end of the sentence, "if you want ... merge your contribution".

Reply:
Addressed in [bbc18b2](https://github.com/vectordotdev/vector/pull/17658/commits/bbc18b285de51e66cd01fcd1c5f89efc71bcfff1)